app/models/crop.rb

```diff
@@ -163,11 +163,11 @@ def planted_from
   # key: plant part (eg. 'fruit')
   # value: count of how many times it's been used by harvests
   def popular_plant_parts
-    popular_plant_parts = Hash.new(0)
-    harvests.each do |h|
-      popular_plant_parts[h.plant_part] += 1 if h.plant_part
-    end
-    popular_plant_parts
```

Comment:
I've changed the structure of what it returns -- but i think it's better now because it's doing the group/count and ordering in the database.
See: 8408fdc845b98721366bb678e1b74e65bf9cdeaa